app/api/domains/osu.py

```diff
@@ -1784,14 +1747,11 @@ async def get_osz(
     if no_video:
         map_set_id = map_set_id[:-1]
 
-    if USING_CHIMU:
-        query_str = f"download/{map_set_id}?n={int(not no_video)}"
-    else:
-        query_str = f"d/{map_set_id}"
+    query_str = f"{map_set_id}?n={int(not no_video)}"
```

Comment:
This can be generalized, no matter whether mirrors support it or not since otherwise this query parameter should be ignored (and is ignored on all mirrors I tested)

Reply:
i don't think it's best practice to program this loosely, but in this context yes it's 99.9_% fine